tests/Unit/Evolution/Systems/CurvedScalarWave/Test_DuDt.cpp

```diff
@@ -94,11 +93,11 @@ calculate_du_dt(const DataVector& used_for_size) {
   auto dt_pi = make_with_value<Scalar<DataVector>>(used_for_size, 0.);
   auto dt_phi = make_with_value<tnsr::i<DataVector, Dim, Frame::Inertial>>(
       used_for_size, 0.);
-  CurvedScalarWave::ComputeDuDt<Dim>::apply(
+  CurvedScalarWave::TimeDerivative<Dim>::apply(
```

Comment:
Please rename this file `Test_DuDt.cpp` -> `Test_TimeDerivative.cpp`

Reply:
done